std/xml.d

```diff
@@ -2854,10 +2855,19 @@ private
 
     void reqc(ref string s, char c)
     {
-        if (s.length == 0 || s[0] != c) throw new TagException("");
+        if (s.length == 0 || s[0] != c)
+            throw new TagException("");
         s = s[1..$];
     }
 
+    char requireOneOf(ref string s, string chars)
+    {
+        if (s.length == 0 || indexOf(chars,s[0]) == -1) throw new TagException("");
```

Comment:
You broke up the wrong line. This is the one longer than 80 columns. The one above was fine.

Instead of `length == 0`, `indexOf` and throwing manually, you could use `empty`, `canFind`, and `enforce`:

```d
        import std.algorithm: canFind;
        import std.exception: enforce;
        enforce(!s.empty && chars.canFind(s[0]), new TagException(""));
```

But I don't mind the current version with `indexOf` etc, if you like it better.